CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
-## [5.0.1] - 2017-05-09
+## 2017-05-09
```

Comment:
remove this line, sorry for the confusion...

We add the date of when it was released not when it was committed since we know that from git history and releases are manually kicked off by maintainers (yes I have raised the issue to automate this).